apps/wallet/src/engine/runners/solana/collectibles.ts

```diff
@@ -92,21 +92,18 @@ export const loadCollectibleMetadata = async (
 		tokenOwner: owner,
 	});
 
-	// TODO: need to resolve fetch metadata of nft using metaplex on mobile
-	if (metadata.jsonLoaded && !metadata.json) {
```

Comment:
@FME849 I wonder if we need to check for the `metadata.jsonLoaded`? I have logged it out and its value is `false` so that there is no fetch happened

Reply:
I checked and found that `metadata.jsonloaded` condition was not necessary, so removing it is fine